index.js

```diff
@@ -47,6 +54,13 @@ var cssGlobbingPlugin = function(options) {
     if(!options.autoReplaceBlock.globBlockContents) options.autoReplaceBlock.globBlockContents = autoReplaceBlockDefaults.globBlockContents;
   }
 
+  if (!(options.scssImportPath instanceof Object)) {
+    throw new gutil.PluginError(PLUGIN_NAME, 'scss import path needs to be an object');
+  }else{
+    if(typeof options.scssImportPath.leading_underscore === 'undefined') options.scssImportPath.leading_underscore = scssImportPathDefaults.leading_underscore;
+    if(typeof options.scssImportPath.filename_extension === 'undefined') options.scssImportPath.filename_extension = scssImportPathDefaults.filename_extension;
+  }
+
```

Comment:
I realize that there is no formal styleguide or anything, but I think spaces after `if` statements (e.g. `if (…`) and around curly braces `} else {` make things much easier to read.